src/pybind/mgr/balancer/module.py

```diff
@@ -663,7 +673,9 @@ def plan_execute(self, plan: str) -> Tuple[int, str, str]:
         if not plan_:
             return (-errno.ENOENT, '', f'plan {plan} not found')
         r, detail = self.execute(plan_)
-        self.update_pg_upmap_activity(plan_)  # update pg activity in `balancer status detail`
+        pg_upmap_activity = cast(bool, self.get_module_option('update_pg_upmap_activity'))
+        if pg_upmap_activity:
```

Comment:
I guess this commit alone would be enough to pacify the issue...
Do we have a plan ti re-enable the command (by default)?

Reply:
This is a new feature in Squid, so I think it makes sense to disable it for now until testing can confirm that it can handle scalability. Then, we can enable it by default.